### DISCUSSION TITLE
Gtag jusqu'à réponse RGPD

### DISCUSSION THREAD

**Discussion by jmradi** [AmauriC/tarteaucitron.js#904] [Q&A]
Hello,

mon client sur https://mindycom.com (tartau plugin pro) souhaite avoir les remontées GA active jusqu'à ce qu'on réponde au pop up tartau… (trop de baisse de fréquentation suite à installation).

Une solution ?

**Comment by AmauriC** [comment]
Hey

Non, ce n'est pas possible, ce ne serait pas conforme.

**Comment by jmradi** [comment]
Pas mal de site le font pourtant… même la CNIL https://www.cnil.fr

**Reply by AmauriC** [reply]
Il n'y a pas GA sur le site de la cnil mais Piwik/Matomo qui lui est ok pour être activé par défaut.
